### PR TITLE
add dark theme

### PR DESCRIPTION
**Pull request by Sneezry**

### CODE REVIEW

File: src/store/Accounts.ts [hunk 1]
```diff
@@ -209,8 +209,7 @@ export class Accounts implements Module {
               return;
             }
             // gen key
-            const randomKey = crypto.getRandomValues(new Uint32Array(30));
-            const wordArray = CryptoJS.lib.WordArray.create(randomKey);
+            const wordArray = CryptoJS.lib.WordArray.random(120);
```
Comment (mymindstorm) [inline src/store/Accounts.ts:212]: I'm looking through the PR that broke this. I will fix this upstream.

**Review by mymindstorm — APPROVED** [review]
Should we change the shadow color?
image

Let's change the color of the dropdowns and checkboxes too.
image

File: _locales/ar/messages.json [hunk 2]
```diff
@@ -255,6 +255,22 @@
         "message": "تباين عالي",
         "description": "Use High Contrast"
     },
```
Comment (mymindstorm) [inline _locales/ar/messages.json:257]: Also, just FYI: you don't need to edit translation files manually. The CI does it for you when you merge to dev.

Comment (Sneezry) [reply _locales/ar/messages.json:257]: Cool! Then we only need update en.json, right?

**Comment by mymindstorm** [comment]
Yep, this PR can stay as is. It won't mess up anythingOn May 19, 2020 8:49 PM, Zhe Li <notifications@github.com> wrote:

@Sneezry commented on this pull request.







In _locales/ar/messages.json:

> @@ -255,6 +255,22 @@

         "message": "تباين عالي",

         "description": "Use High Contrast"

     },



Cool! Then we only need update en.json, right?

File: sass/popup.scss [hunk 3]
```diff
@@ -586,8 +586,67 @@ svg {
...
+    background: none;
+    width: 34px;
+    height: 16px;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    outline: none;
+  }
+
+  .prefrence-item input[type="checkbox"]::before {
```
Comment (mymindstorm) [inline sass/popup.scss:617]: The fact that you can do this in pure CSS is really cool

Comment (Sneezry) [reply sass/popup.scss:617]: 😄

**Pull request merged**